src/Medidata.ZipkinTracer.Core/ZipkinClient.cs

```diff
@@ -20,11 +17,14 @@ public class ZipkinClient : ITracerClient
         private string requestName;
         private ITraceProvider traceProvider;
         private ILog logger;
+        private List<string> internalDomainList;
 
         public ZipkinClient(ITraceProvider tracerProvider, string requestName, ILog logger) : this(tracerProvider, requestName, logger, new ZipkinConfig(), new SpanCollectorBuilder()) { }
 
         public ZipkinClient(ITraceProvider traceProvider, string requestName, ILog logger, IZipkinConfig zipkinConfig, ISpanCollectorBuilder spanCollectorBuilder)
         {
+            internalDomainList = zipkinConfig.GetInternalDomainList();
```

Comment:
@bvillanueva-mdsol maybe we had better "zipkin" prefix in the key name... as some app has a lot of keys in appsettings.